docs/reference/docs/bulk.asciidoc

```diff
@@ -219,7 +219,18 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=source_excludes]
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=source_includes]
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeout]
+`timeout`::
++
+--
+(Optional, <<time-units, time units>>)
+Period each action waits for the following operations:
+
+* <<index-creation,Automatic index creation>>
+* <<dynamic-mapping,Dynamic mapping>> updates
+* <<index-wait-for-active-shards,Waiting for active shards>>
+
+The timeout applies to each operation separately. Defaults to `1m` (one minute).
```

Comment:
While correct, I think we should define this a bit more loosely to be able to change this without calling it a breaking change. Something like:

```
Defaults to `1m` (one minute). This guarantees that {es} will wait for at least the timeout before failing, but the actual wait time could be longer (in particular when multiple waits occur).
```